hopsworks-web/yo/app/scripts/controllers/featureViewInfoCtrl.js

```diff
@@ -62,13 +63,48 @@ angular.module('hopsWorksApp')
                 self.scalaCode = self.getScalaCode(self.feature)
             };
 
-            /**
-             * Closes the modal
-             */
-            self.close = function () {
-                $uibModalInstance.dismiss('cancel');
-            };
+            self.isToggled = function(feature) {
+                if(!self.selectedFeature || !feature) {
+                    return false;
+                } else {
+                    return self.selectedFeature.featuregroup.id === feature.featuregroup.id && self.selectedFeature.name === feature.name && self.tgState === true;
+                }
+            }
+
+            self.toggle = function(feature) {
```

Comment:
In general this `if`can be simplified. 
```
return !(self.selectedFeature.featuregroup.id === feature.featuregroup.id && self.selectedFeature.name === feature.name && self.tgState === true)
```
return `false` if all the condition above are met, true otherwise.